src/swap/types.ts

```diff
@@ -72,11 +84,13 @@ type LifecycleStatusDataShared = {
 export type LifeCycleStatus =
   | {
       statusName: 'init';
-      statusData: LifecycleStatusDataShared;
+      statusData: LifecycleStatusDataBase;
```

Comment:
I know the word Base here is correct, but I would like to avoid using it, so there is no confusion with Base chain.